### DISCUSSION TITLE
Installing mods

### DISCUSSION THREAD

**Discussion by R3DD3VIL1982** [Ch4r0ne/ARK-Ascended-Server-Manager#41] [General]
Hi i can only seem to install winter wonderland whats the usual process for installing mods. TIA

**Comment by R3DD3VIL1982** [comment]
All the searches online say to just ad the mod ID to the command line, witch happens when you type the mod ID in the mod box on the ARK Ascended Server Manager.
you can have 4 different mod IDs in the command line but there is only ever 1 mod actually running.
iv tried . iv tried - and = to separate the mod IDs but i still cant get more then 1 mod to run at a time.
Thanks in advance for any help

**Comment by KestrelKitezh** [comment]
isnt it mod=0999999,90991929,0999 ?
